components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1017,6 +1022,14 @@ interface JsonRpcService {
                        string address,
                        string chain_id) => (string balance, ProviderError error, string error_message);
 
+  // Fetches the balances of multiple ERC20 tokens for an address using a
+  // BalanceScanner contract
+  GetERC20TokenBalances(array<string> contracts,
```

Comment:
Please document here by linking to the list of chains supported in wallet_constants and mention this API only support these chains.